src/export_data.py

```diff
@@ -0,0 +1,122 @@
+"""Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+import argparse
+import ee
+from utils import TaskManager, SampleExporter, stack_bands_from_imagery
+from utils import RoadImagery, MultiSpectralImagery, TropomiImagery
+from utils import DSMImagery, WindImagery
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Export Data from EarthEngine'
+                                     )
+    parser.add_argument('--params_path', type=str,
```

Comment:
It would be helpful to add what format/fields we expect in the json parameters and what good defaults would be. This could be in the form of the help string or a docstring. It looks like this file depends on a lot of parameters from the json parameters file.